examples/config files - basic/connector-ldap.yml

```diff
@@ -22,6 +22,15 @@ password: "LDAP password goes here"
 host: "ldap://ldap.example.com"
 base_dn: "DC=example,DC=com"
 
+# (optional) You can specify what Authentication method to bind LDAP
+# connection with. You can choose either Anonymous, Simple or NTLM.
+# If username is not specified above, authentication method will default to anonymous
```

Comment:
I think it's important to use the work "override" here since that is what the connector is doing.  So the note would be something like `If username is not specified above, the LDAP connector will override the auth method and set it to 'anonymous'`

I also think it is important to the note that this line replaced - `If you choose anonymous, you don't have to specify username and password above.`.